packages/@react-spectrum/button/src/ToggleButton.tsx

```diff
@@ -51,6 +51,7 @@ export const ToggleButton = React.forwardRef(function ToggleButton(props: Spectr
       <button
         {...styleProps}
         {...mergeProps(buttonProps, hoverProps)}
+        type="button"
```

Comment:
I think this is the default in `useButton`? Do we need to set it here?

Reply:
you think set it in useToggleButton would be better? i think that'd be fine

Reply:
oh yeah, i guess we don't need to set it here